src/Nordea/Resources/Colors.elm

```diff
@@ -11,17 +12,21 @@ module Nordea.Resources.Colors exposing
     , grayDarkest
     , grayEclipse
     , grayLight
+    , grayLight70
     , grayMedium
     , grayNordea
     , grayWarm
     , green
+    , green70
     , greenDark
     , red
+    , red70
```

Comment:
Could these new colors get more descriptive names like the other colors?

Reply:
Great idea! 🎉